tests/lib/rules/await-async-query.test.ts

```diff
@@ -14,21 +14,31 @@ const ruleTester = createRuleTester();
 interface TestCode {
   code: string;
   isAsync?: boolean;
+  testingFramework: string;
 }
 
-function createTestCode({ code, isAsync = true }: TestCode) {
+function createTestCode({ code, isAsync = true, testingFramework }: TestCode) {
+  if (!testingFramework) {
+    return `
+      import { render } from '@testing-library/react'
+      test("An example test",${isAsync ? ' async ' : ' '}() => {
+        ${code}
+      })
+    `;
+  }
   return `
-    import { render } from '@testing-library/react'
-    test("An example test",${isAsync ? ' async ' : ' '}() => {
-      ${code}
-    })
-  `;
+      import { render } from '@marko/testing-library'
```

Comment:
I guess this should be

```suggestion
      import { render } from '${testingFramework}'
```

Reply:
done